tests/integration/profiling/rule_based_profilers/conftest.py

```diff
@@ -0,0 +1,1512 @@
+import datetime
+import os
+import shutil
+from typing import Dict, List, Optional
+
+import numpy as np
+import pandas as pd
+import pytest
+from freezegun import freeze_time
+from ruamel.yaml import YAML
+
+from great_expectations import DataContext
+from great_expectations.core import ExpectationConfiguration, ExpectationSuite
+from great_expectations.data_context.util import file_relative_path
+from great_expectations.datasource.data_connector.util import (
+    get_filesystem_one_level_directory_glob_path_list,
+)
+from great_expectations.rule_based_profiler.config.base import (
+    ruleBasedProfilerConfigSchema,
+)
+from tests.conftest import skip_if_python_below_minimum_version
+
+yaml = YAML(typ="safe")
+
+
+@pytest.fixture
+def alice_columnar_table_single_batch_context(
+    monkeypatch,
+    empty_data_context,
+    alice_columnar_table_single_batch,
+):
+    skip_if_python_below_minimum_version()
```

Comment:
Do you have any ideas on how to get rid of this pattern? We added this check to basically skip the test if the version was below 3.7 (due to dataclasses) but there has to be a better way to do this.

Instead of explicitly adding this to each test, we should be able to modify something at the `conftest` level. Not a blocker for this PR but I'd really love to see a more idiomatic pattern here.